arch/arm64/src/common/arm64_cpustart.c

```diff
@@ -226,6 +222,12 @@ int up_cpu_start(int cpu)
 
 void arm64_boot_secondary_c_routine(void)
 {
+  struct tcb_s *tcb = current_task(this_cpu());
```

Comment:
I think most places use this_task(), does it not work here ?

Reply:
it not work here, 
 Because we need to complete the initialization percpu before using percpu, and this is where the initialization takes place.